lib/src/room.dart

```diff
@@ -646,12 +736,17 @@ class KurentoRoom extends ChangeNotifier implements Room {
         }
         break;
 
-      // All of the other status values -- END, CANCEL, etc. -- are end states.
-      default:
+      case 'END':
+      case 'CANCEL':
```

Comment:
Did you double check all the possible event to get here? I would like to be sure we haven't forgotten something.

Reply:
i checked with peter and there is no other event that finish a call